src/test/java/redis/clients/jedis/commands/jedis/BitCommandsTest.java

```diff
@@ -153,13 +155,20 @@ public void bitposModifier() {
     assertEquals(1, jedis.bitpos("mykey", true, BitPosParams.bitPosParams()));
     assertEquals(18, jedis.bitpos("mykey", true, BitPosParams.bitPosParams().start(2)));
     assertEquals(18, jedis.bitpos("mykey", true, BitPosParams.bitPosParams().start(2).end(-1)));
+  }
+
+  @Test
+  @SinceRedisVersion(value = "7.0.0", message = "7.0.0 Added the BYTE|BIT option.")
+  public void bitposModifierByte() {
+    jedis.set("mykey", "\\x00\\xff\\xf0");
```

Comment:
The name modifier suggests the above test is specifically for BYTE|BIT. So just

```suggestion
```

and put the condition above.